signing_service/setup.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python3
+
+import os
+from setuptools import setup
+
+
+def get_version():
+    with open(
+        os.path.join(
+            os.path.dirname(os.path.dirname(os.path.abspath(__file__))),
+            'RELEASE-VERSION',
+        ),
+        'r'
+    ) as version_file:
+        return version_file.read()
+
+
+setup(
+    name='Signing-Service',
+    version=get_version(),
```

Comment:
Anything related to `setup.py` and requirements should really be in a separate commit. It's not related in any way to other changes.

Reply:
:+1: done